viewer/packages/360-images/src/collection/Image360Collection.ts

```diff
@@ -57,4 +68,9 @@ export interface Image360Collection {
    * Assign a default style which affects all annotations
    */
   setDefaultAnnotationStyle(appearance: Image360AnnotationAppearance): void;
+
+  /**
+   * Find a 360 image associated with a asset with the given assetRef
+   */
+  findAsset(assetRef: IdEither): Promise<Image360AnnotationAssetQueryResult[]>;
```

Comment:
Should we consider future-proofing this a bit, and maybe adding support for lookups on label? I'm thinking that it'd be good if we can add FDM search without adding new functions, maybe something like:
```
  findImage(filter: AssetFilter | LabelFilter ): Promise<Image360AnnotationAssetQueryResult[]>;
```
?